ledger-core/virtual/execute/execute.go

```diff
@@ -131,6 +131,10 @@ func (s *SMExecute) migrationDefault(ctx smachine.MigrationContext) smachine.Sta
 }
 
 func (s *SMExecute) Init(ctx smachine.InitializationContext) smachine.StateUpdate {
+	if s.pulseSlot.State() != conveyor.Present {
```

Comment:
No need to check it here, SMExecute isn't handler and can only be spawned in handler :)